### PR TITLE
BugFix: Bug Fixes from PED

### LINKED ISSUE #345: [PE-D][Tester B] `import` command not responding when parameter contains the `"` character
## Steps to reproduce

1. Run the application
1. Enter the command `import i|"`

## Expected

The following message is displayed:

```
File " is not a csv file 
Import command only accepts csv files
```

## Actual

No message is displayed.

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: chin-herng/ped#1

### LINKED ISSUE #348: [PE-D][Tester B] `import` command not responding when importing an empty csv file
## Steps to reproduce

1. Create an empty file named `a.csv` in `C:\Users\USER\Desktop\CS2103T\New folder`
1. Run the application
1. Enter the command `import i|C:\Users\USER\Desktop\CS2103T\New folder\a.csv`

## Expected

The following message is displayed:

```
Import completed with errors

No valid persons were found. Csv file is empty or error occurred reading from csv file
Successful imports: 0
Unsuccessful imports: 0
```

## Actual

No message is displayed.

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: chin-herng/ped#2

### LINKED ISSUE #359: [PE-D][Tester E] Incorrect error message for importing person
## Description of the Bug
Typos found in displayed error message for `import`

## Steps to Reproduce
1. `import`

## Expected behavior
```
Invalid command format! 
import: Imports persons from specified filepath. Must be an absolute CSV file path
Parameters: i|FILEPATH
Example: import i|C:usr/lib/text.csv
```

## Actual behavior
```
Invalid command format! 
import: Imports persons from specified filepath. Must be an absolute CSV file path
Parameters: filePath
[i|import]
Example: importi|C:usr/lib/text.csv
```

## Additional comments
Unsure if there's a need for the prefix `i|` as the only parameter currently is the file path.

-------------
Labels: `type.FeatureFlaw` `severity.VeryLow`
original: tsulim/ped#3

### LINKED ISSUE #362: [PE-D][Tester C] Able to add new person when there are repetitive names under csv file
## Describe the bug
When there are duplicate columns in the CSV file, I am able to add a person with a column of 2 emails of differing emails. However, if both the columns have the same email, I am unable to add the person

## To Reproduce
Have 2 email columns with the same email or different email.
## Expected behavior
Should not be able to add the person.

## Screenshots/Video

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: headcube1/ped#10

### PR DESCRIPTION
**Pull request by Pughal77**
Bug fix from PED

1) Now import Command properly displays its Usage message, with
a whitespace between import and i|

fixes #359 

2) ImportCommandParser now also checks for invalid characters in
filePath which is already caught by the InvalidPathException in
java

fixes #345 

3) Bug fix for when csv file to be imported is empty

Currently the readCsvFile method in CsvUtil only checks if the values
are empty but not if the rows themselves are empty. Now, it is
handled by readCsvFile to return an empty nullableData and
empty report

fixes #348

4) Bug fix for duplicate headers when they have the same value

For the case of duplicate headers, there is a bug in which when
there are 2 email coloumns with the same value, the person is not
added to the addressBook. (As stated in the PE-D, issue #362).

This is fixed by now correctly ignoring the duplicate column

fixes #362

### CODE REVIEW

**Review by Zer0Legion — COMMENTED** [review]
Great bug fixes!

File: src/main/java/seedu/address/commons/util/CsvUtil.java [hunk 1]
```diff
@@ -69,13 +67,17 @@ public static Pair<Optional<List<Map<String, String>>>, String> readCsvFile(
...
         try {
             CSVReader reader = new CSVReaderBuilder(new FileReader(filePath.toString())).build();
-            List<String> headers = List.of(reader.readNext()); // first line should be headers
-            List<Integer> columnsToSkip = columnsToSkip(headers, compulsoryParameters, optionalParameters);
-            List<String[]> rows = reader.readAll();
-            Pair<List<Map<String, String>>, String> result = parseData(rows, headers, columnsToSkip);
-            data = result.getKey();
-            errorMsgs = result.getValue();
-            nullableData = Optional.of(data);
+            Optional<String[]> headers = Optional.ofNullable(reader.readNext()); // first line should be headers
```
Comment (jayllo-c) [inline src/main/java/seedu/address/commons/util/CsvUtil.java:70]: Nice use of optional<>.

**Review by jayllo-c — COMMENTED** [review]
Nice fixes

**Review by delishad21 — COMMENTED** [review]
Good bugfixes. This will help to maintain the user experience of our application by preventing confusing app responses

**Pull request merged**